.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+install:
+  - pip install -r requirements.txt
+  - python setup.py install
+  - pip install pep8
+script: 
+  - ./test/tsht
+  - pep8 hocr-*
```

Comment:
I will try to look and test it further in the next days, also your other PRs. I guess the order in which these PR are merged will not depend that much, right?